### PR TITLE
Fix regression of JsonAmm hotfix

### PR DESCRIPTION
**Pull request by Bladehawke**
Also added a test to help catch future regressions in this area.  I probably should have put `bin/x64/plugins/cyber_engine_tweaks/mods/AppearanceModMenu/` into a constant for readability....

### CODE REVIEW

File: test/unit/mods.example.ts [hunk 1]
```diff
@@ -1656,6 +1656,51 @@ export const FallbackForNonMatchedAndInvalidShouldPromptForInstall = new Map<
...
+      proceedLabel: InstallChoices.Proceed,
+      proceedOutInstructions: [
+        {
+          type: "copy",
+          source: path.join(ARCHIVE_PREFIX, "proximas_propshop_v4.archive"),
+          destination: path.join(ARCHIVE_PREFIX, "proximas_propshop_v4.archive"),
+        },
+        {
+          type: "copy",
+          source: path.normalize(
```
Comment (effs) [inline test/unit/mods.example.ts:1684]: I added a couple helpers for the expected instructions: https://github.com/E1337Kat/cyberpunk2077_ext_redux/blob/a7f5728c7b10dc43dd91f50206947ad350604eb2/test/unit/utils.helper.ts#L41

Comment (Bladehawke) [reply test/unit/mods.example.ts]: Thanks, I hadn't noticed that :)

**Comment by Bladehawke** [comment]
Sorry, my morning appointments took several hours longer than I expected.

**Review by E1337Kat — APPROVED** [review]
lg2m

**Pull request merged**